manifests/install/charts/as-a-second-scheduler/Chart.yaml

```diff
@@ -15,7 +15,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.21.6
+version: 0.21.7
```

Comment:
Can we make it 0.21.6-1? (indicating the scheduler image isn't bumped). Also change appVersion to 0.21.6-1?